src/GameState/Marble.tsx

```diff
@@ -4,28 +4,29 @@ import React from 'react';
 const MARBLE_COLOR_RADIAL = ['warmGray.300', 'black', 'white', 'red.700'];
 
 export const Marble = (props: any) => {
-
-
   const size = props.size;
-  const child = <Circle size={size} m={0} p={0} bgColor={getMarbleColor(props.value)} />;
+  const child = (
+    <Circle size={size} m={0} p={0} bgColor={getMarbleColor(props.value)} />
+  );
 
   const marbleClick = async () => {
```

Comment:
We generally use the keyword handle: `handleClickMarble` when the method is linked to an event.